src/Tickets/Flexible_Tickets/Base.php

```diff
@@ -284,4 +312,67 @@ public function filter_attendees_event_details_top_label( string $label, int $po
 		// This controller will not register if ECP is not active: we can assume we'll have ECP translations available.
 		return __( 'Series', 'tribe-events-calendar-pro' );
 	}
+
+	/**
+	 * Filters the columns displayed in the Series editor events List.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string,string> $columns The list of columns to filter.
+	 *
+	 * @return array<string,string> The filtered list of columns.
+	 */
+	public function filter_series_editor_occurrence_list_columns( array $columns ): array {
+		return Tribe__Main::array_insert_before_key( 'actions',
+			$columns,
+			[
+				'ticket_types' => esc_html__( 'Attached Ticket Types', 'event-tickets' )
+			]
+		);
+	}
+
+	/**
+	 * Renders the content of the "Attached Ticket Types" column in the Series editor events List.
+	 *
+	 * @since TBD
+	 *
+	 * @param Occurrence $occurrence
+	 *
+	 * @return void
+	 */
+	public function render_series_editor_occurrence_list_column_ticket_types( Occurrence $occurrence ) {
+		$event_id = $occurrence->post_id;
+		$tickets  = Tickets::get_event_tickets( $event_id );
+
+		if ( empty( $tickets ) ) {
+			echo '&mdash;';
+			return;
+		}
+
+		$has_rsvp        = false;
+		$has_ticket      = false;
+		$has_series_pass = false;
+
+		foreach ( $tickets as $ticket ) {
+			// check if RSVP available.
+			if ( $ticket->provider_class === RSVP::class ) {
+				$has_rsvp = true;
+				continue;
+			}
+			// check if ticket type is series pass or general ticket.
+			if ( get_post_meta( $ticket->ID, '_type', true ) === Series_Passes::TICKET_TYPE ) {
+				$has_series_pass = true;
+			} else {
+				$has_ticket = true;
+			}
+		}
+
+		$admin_views = new Admin_Views;
+		$admin_views->template( 'ticket-types-column/types', [
+			'has_rsvp'        => $has_rsvp,
```

Comment:
Following my comment in the template, I would make this an array of tickets by type.